test/pipelines/test_eval.py

```diff
@@ -2100,6 +2110,7 @@ def test_load_legacy_evaluation_result(tmp_path):
     assert "content" not in eval_result["legacy"]
 
 
+@pytest.mark.unit
 def test_load_evaluation_result_w_none_values(tmp_path):
```

Comment:
If this is simply testing that we can load results with `None`s in them, can this test be simplified? It seems overly complex for its name.

Reply:
I believe it is to generally test the `load_evaluation` feature in general. I can update the test name to reflect that.